### PR TITLE
messageLogger: apply deleted style to ephemeral messages

### PR DESCRIPTION
**Pull request by VMGuy23**
The bot "thinking" text was previously not shown red when deleted.
I noticed this bug when using YAGPDB's poll feature. When the poll is created, a "YAGPDB.xyz is thinking..." during the creation of the poll, and deleted after. However, it is not shown in red once deleted. Image below.
image
This PR should fix this bug.

### CODE REVIEW

**Comment by MrDiamondDog** [comment]
That's because it's not an actual message. It is a placeholder for when the bot is loading it's response. It can not be changed, and will always say `(Bot name) is thinking...`.

**Comment by VMGuy23** [comment]
> That's because it's not an actual message. It is a placeholder for when the bot is loading it's response. It can not be changed, and will always say `(Bot name) is thinking...`.

That doesn't stop it from being styled red to show that it is deleted. The right-click menu still correctly shows the options for a deleted message

**Comment by VMGuy23** [comment]
why is the check only running occasionally

**Comment by Vendicated** [comment]
since you are a first time contributor, i have to manually approve workflow runs https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

File: src/plugins/messageLogger/deleteStyleText.css [hunk 1]
```diff
@@ -3,6 +3,11 @@
     color: #f04747 !important;
 }
 
+/* Bot "thinking" text highlighting */
+.messagelogger-deleted [class*="colorStandard-"] :is(div) {
```
Comment (Vendicated) [inline src/plugins/messageLogger/deleteStyleText.css:7]: this :is() is obsolete, since there is only one selector. :is is used to group multiple selectors

Comment (VMGuy23) [reply src/plugins/messageLogger/deleteStyleText.css]: Should  be fixed now

**Comment by Vendicated** [comment]
thank you

**Pull request merged**